### ISSUE TITLE
Home Page: SC 2.4.13 (AAA): Keyboard focus meet minimum requirements for appearance.

### ISSUE THREAD

**Issue by BobTorgerson** [ua-snap/nccwsc-projects#160]
**Location(s):**
This issue is consistent throughout the page.

**Reasons flagged:**
- Focus is not at least as large as the area of a 2 CSS pixel thick perimeter of the unfocused component or sub-component.
- Focus does not have a contrast ratio of at least 3:1 between the same pixels in the focused and unfocused states.
- Focus appearance is inconsistent throughout the page.

**Comment by hollychandler** [comment]
Approved!

**Comment by BobTorgerson** [comment]
It appears this might have been a misinterpretation. This was a design element specifically requested by Holly to have a "glow" effect around the text field, not a focus on the search field.
